internal/build/custom_builder.go

```diff
@@ -83,10 +80,18 @@ func (b *CustomBuilder) Build(ctx context.Context, refs container.RefSet,
 	if expectedBuildResult != nil {
 		extraEnvVars = append(extraEnvVars,
 			fmt.Sprintf("EXPECTED_REF=%s", container.FamiliarString(expectedBuildResult)))
+		extraEnvVars = append(extraEnvVars,
+			fmt.Sprintf("EXPECTED_IMAGE=%s", reference.Path(expectedBuildResult)))
```

Comment:
Is `EXPECTED_IMAGE` misleading?

Technically, in `distribution/reference` parlance, this is the "path", but I think `EXPECTED_PATH` is extremely vague/confusing.

I almost went with `EXPECTED_IMAGE_NAME` but wasn't sure if that was just redundantly & long 🤷

Reply:
I think `EXPECTED_IMAGE` is fine. Is the path always part of the image name though? I'm thinking of the ECR case (https://docs.tilt.dev/personal_registry.html#elastic-container-registrys-repository-dance).

Reply:
Thanks for all the research & thoughts! Will keep it as `EXPECTED_IMAGE` - I'll make sure when adding to custom build docs that I give an examples of values for each of the env vars, which should help